src/test/java/com/epam/tc/hw3/pages/AbstractPage.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.tc.hw3.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public abstract class AbstractPage {
+    protected WebDriver driver;
+    protected static final String HOMEPAGE_URL = "https://jdi-testing.github.io/jdi-light/index.html";
+    protected final int waitTimeoutSeconds = 10;
+
+    protected abstract void openPage(
```

Comment:
It looks like the method is not overridden in all the subclasses, check if it's really required to keep it